canonical-websites/www.canonical.com
# Tidy up legacy styling

## Done
Replaced any relative image paths to asset server URLs. Tidied up some old styles and removed unused styles. Also fixed a slight alignment issue on the header.

## QA
- Pull down branch
- Run `make run`
- Click through the site and check everything looks the same
- Check that the Canonical logo aligns with the global nav

## Details
Fixes https://github.com/canonical-websites/www.canonical.com/issues/56, https://github.com/canonical-websites/www.canonical.com/issues/76
